docs/en/plugins/README.md

```diff
@@ -0,0 +1,413 @@
+Loomio’s recently released a plugin architecture, which will allow you quickly and easily add functionality to Loomio on your own instance. There’s some good docs in (plugin README), but I thought it might be helpful to step through how we make plugins here, step by step
+
+#### WHAT WE’LL BE MAKING:
+
+The thing we’ll be making this time is a tagging plugin, which will allow group coordinators to specify tags, which can then be used to classify discussions which are related, but distinct (Think something like ‘Board meeting minutes’, or ‘The Party Planning Committee’)
+
+It'll look like this:
+
+![](images/a_tag_yay.png)
+
+By the end of this article, we'll have
+
+- Added a new database table and model to the Loomio rails app
+- Added a new controller, route, and serializer to expose an API endpoint
+- Modified existing Loomio classes to create model relationships
+- Added models and record stores for storing data on the client
+- Created an angular directive which shows our tags on the dashboard
+- Created an angular directive which displays our tags in the view
+- Added testing functionality to our rails controller which can be hooked up to a CI or run locally
+
+We won't have
+
+- Created a form to allow group admins to add, edit or destroy tags
+- Made another directive to make the tags appear on the thread page
+
+But, you can check out the repository for this plugin [on github](http://github.com/loomio/loomio_tags) to see our most up-to-date progress.
+
+Let's get started!
+
+### PART 1: Create a basic plugin.
+First off, we’re going to create a local plugin, and initialize it with a git repository so it’s separate from the Loomio core codebase. This guide assumes you have a local dev instance of Loomio up and running
+
+First, create a new folder in the `/plugins` folder with the name of your plugin
+```
```

Comment:
First, create a new folder in the `/plugins` folder with the name of your plugin**:** [add colon]